content/en/agent/docker/apm.md

```diff
@@ -83,7 +83,7 @@ List of all environment variables available for tracing within the Docker Agent:
 | `DD_APM_NON_LOCAL_TRAFFIC` | Allow non-local traffic when [tracing from other containers](#tracing-from-other-containers).                                                                                                                                                                                                                                                        |
 | `DD_APM_IGNORE_RESOURCES`  | Configure resources for the Agent to ignore. Format should be comma separated, regular expressions. Example: <code>GET /ignore-me,(GET\|POST) /and-also-me</code>.                                                                                                                                                                                       |
 | `DD_APM_ANALYZED_SPANS`    | Configure the spans to analyze for transactions. Format should be comma separated instances of <code>\<SERVICE_NAME>\|;\<OPERATION_NAME>=1</code>. i.e. <code>my-express-app\|;express.request=1,my-dotnet-app\|;aspnet_core_mvc.request=1</code>. You can also [enable it automatically][3] with the configuration parameter in the Tracing Client. |
-| `DD_APM_ENV`               | Sets the default [environment][4] for your traces.                                                                                                                                                                                                                                                                                                   |
+| `DD_APM_ENV`               | Sets the default [environment][4] for your traces. If possible, it is recommended to define the environment [through the tracer][5].                                                                                                                                                                                                                                                                                                    |
```

Comment:
should we note that this is not needed if `DD_ENV` is set?

Reply:
Yes actually we want to remove this - thanks!